### PR TITLE
fix(sui-i18n): fixed a bug related with regex

### PR DESCRIPTION
**Pull request by carlesnunez**
## Description
I've changed the REGEX in order to fix an edge case that was throwing errors. 

I've also increased our test suite in order to avoid future problems regarding slugify function.

### CODE REVIEW

File: packages/sui-i18n/src/slugify.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export function slugify(str) {
   return str
     .toLowerCase()
     .replace(/.{1}/g, replaceCharIfNeeded)
-    .replace(/[^a-z0-9 - .]/g, '') // remove invalid chars only letters numbers, dashes and dots
+    .replace(/[^a-z0-9. -]/g, '') // remove invalid chars only letters numbers, dashes and dots
```
Comment (carlosvillu) [inline packages/sui-i18n/src/slugify.js]: You have to escape the dot

Because dot means all characters
